docs/en/relying-party-solution.rst

```diff
@@ -111,10 +184,11 @@ In the **Same Device Flow** the Relying Party uses a HTTP response redirect (sta
     HTTP/1.1 /pre-authz-endpoint Found
     Location: https://wallet-providers.eudi.wallet.gov.it?
     client_id=https%3A%2F%2Frelying-party.example.org%2Fcb
-    &request_uri=https%3A%2F%2Frelying-party.example.org%2Frequest_uri%3Fid%3Drandom-value
+    &request_uri=https%3A%2F%2Frelying-party.example.org%2Frequest_uri
+    &discovery_uri=https%3A%2F%2Frelying-party.example.org%2Fdiscovery_uri
 
 
-In the **Cross Device Flow**, a QR Code is shown by the Relying Party to the User in order to provide the Authorization Request. The User frames the QR Code using their Wallet Instance. The QR Code payload MUST be a **Base64 encoded string**.
+In the **Cross Device Flow**, a QR Code is shown by the Relying Party to the User in order to provide the Authorization Request. The User frames the QR Code using their Wallet Instance.
```

Comment:
I have some doubts here. Base64 was introduced because if the QR code is mistakenly scanned with the camera app (as usually happens) the presence of a URL like `haip://` could trigger the opening of some unwanted deep link or cause the appearance an error message on the OS side. So my advice is to always leave the encoding in `base64` so that the QR code can be interpreted correctly exclusively by the wallet instance.
​

Reply:
During the workshop in Rome we have faced this issue

@torsten and Paul Bastien pointed out to us that it is necessary that the QRcode payload must be in clear text and not base64, otherwise we lose in terms of interoperability with other wallet solutions

Reply:
I don't think base64 creates an interoperability problem if everyone follows the same rules. How do we plan to solve the problem that anyone could register that deep link and therefore the operating system could redirect traffic to apps that are not wallet instances? If we want something textplain (which is also compatible with the same device flow) I aim for a universal link that is managed by a central entity (like trust anchor)

Reply:
according to our chat, it seems that this aspect is now resolved with the achievement of the agreement that the QRCode payload is in plain text and not base64 encoded and with the usage of the universal links instead of the custom url scheme